pandas_gbq/load.py

```diff
@@ -60,24 +61,35 @@ def load_chunks(
     job_config.source_format = "CSV"
     job_config.allow_quoted_newlines = True
 
-    if schema is None:
+    # Explicit schema? Use that!
+    if schema is not None:
+        schema = pandas_gbq.schema.remove_policy_tags(schema)
+        job_config.schema = pandas_gbq.schema.to_google_cloud_bigquery(schema)
+    # If not, let BigQuery determine schema unless we are encoding the CSV files ourselves.
+    elif not FEATURES.bigquery_has_from_dataframe_with_csv:
         schema = pandas_gbq.schema.generate_bq_schema(dataframe)
+        schema = pandas_gbq.schema.remove_policy_tags(schema)
+        job_config.schema = pandas_gbq.schema.to_google_cloud_bigquery(schema)
 
-    schema = pandas_gbq.schema.add_default_nullable_mode(schema)
+    chunks = split_dataframe(dataframe, chunksize=chunksize)
+    for remaining_rows, chunk in chunks:
+        yield remaining_rows
 
-    job_config.schema = [
-        bigquery.SchemaField.from_api_repr(field) for field in schema["fields"]
-    ]
-
-    chunks = encode_chunks(dataframe, chunksize=chunksize)
-    for remaining_rows, chunk_buffer in chunks:
-        try:
-            yield remaining_rows
-            client.load_table_from_file(
-                chunk_buffer,
+        if FEATURES.bigquery_has_from_dataframe_with_csv:
```

Comment:
Not strictly necessary (just the "omit policyTags" logic was), but I thought this might be a good opportunity to use more logic from `google-cloud-bigquery`, per https://github.com/pydata/pandas-gbq/issues/339

The CSV encoding in `google-cloud-bigquery` is still relatively new, so I didn't want to bump our minimum `google-cloud-bigquery` versions yet. Discussion: https://github.com/pydata/pandas-gbq/issues/357